[gmp] Build errorr

Package: gmp:x86-windows-static -> 6.2.1#20

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32820.0
-    vcpkg-tool version: 2023-06-22-f19f3d9939100085fe2f7e41502d60b08e2083b6
    vcpkg-scripts version: 5ac89392d 2023-07-11 (4 hours ago)

**To Reproduce**

`vcpkg install gmp`

**Failure logs**

-- Downloading https://mirror.msys2.org/msys/x86_64/automake1.16-1.16.5-1-any.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/automake1.16-1.16.5-1-any.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/automake1.16-1.16.5-1-any.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/automake1.16-1.16.5-1-any.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/automake1.16-1.16.5-1-any.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/automake1.16-1.16.5-1-any.pkg.tar.zst -> msys2-automake1.16-1.16.5-1-any.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/libiconv-1.17-1-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/libiconv-1.17-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libiconv-1.17-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libiconv-1.17-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libiconv-1.17-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libiconv-1.17-1-x86_64.pkg.tar.zst -> msys2-libiconv-1.17-1-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/libintl-0.21-2-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/libintl-0.21-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libintl-0.21-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libintl-0.21-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libintl-0.21-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libintl-0.21-2-x86_64.pkg.tar.zst -> msys2-libintl-0.21-2-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/zlib-1.2.13-1-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/zlib-1.2.13-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/zlib-1.2.13-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/zlib-1.2.13-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/zlib-1.2.13-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/zlib-1.2.13-1-x86_64.pkg.tar.zst -> msys2-zlib-1.2.13-1-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/tar-1.34-3-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/tar-1.34-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/tar-1.34-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/tar-1.34-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/tar-1.34-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/tar-1.34-3-x86_64.pkg.tar.zst -> msys2-tar-1.34-3-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/findutils-4.9.0-3-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/findutils-4.9.0-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/findutils-4.9.0-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/findutils-4.9.0-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/findutils-4.9.0-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/findutils-4.9.0-3-x86_64.pkg.tar.zst -> msys2-findutils-4.9.0-3-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/gmp-6.2.1-2-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/gmp-6.2.1-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/gmp-6.2.1-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/gmp-6.2.1-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/gmp-6.2.1-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/gmp-6.2.1-2-x86_64.pkg.tar.zst -> msys2-gmp-6.2.1-2-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/gcc-libs-11.3.0-3-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/gcc-libs-11.3.0-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/gcc-libs-11.3.0-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/gcc-libs-11.3.0-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/gcc-libs-11.3.0-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/gcc-libs-11.3.0-3-x86_64.pkg.tar.zst -> msys2-gcc-libs-11.3.0-3-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/libbz2-1.0.8-4-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/libbz2-1.0.8-4-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libbz2-1.0.8-4-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libbz2-1.0.8-4-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libbz2-1.0.8-4-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libbz2-1.0.8-4-x86_64.pkg.tar.zst -> msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/liblzma-5.4.1-1-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/liblzma-5.4.1-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/liblzma-5.4.1-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/liblzma-5.4.1-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/liblzma-5.4.1-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/liblzma-5.4.1-1-x86_64.pkg.tar.zst -> msys2-liblzma-5.4.1-1-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/libzstd-1.5.4-1-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/libzstd-1.5.4-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libzstd-1.5.4-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libzstd-1.5.4-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libzstd-1.5.4-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libzstd-1.5.4-1-x86_64.pkg.tar.zst -> msys2-libzstd-1.5.4-1-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/libreadline-8.2.001-3-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/libreadline-8.2.001-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libreadline-8.2.001-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libreadline-8.2.001-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libreadline-8.2.001-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libreadline-8.2.001-3-x86_64.pkg.tar.zst -> msys2-libreadline-8.2.001-3-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/mpfr-4.2.0-2-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/mpfr-4.2.0-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/mpfr-4.2.0-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/mpfr-4.2.0-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/mpfr-4.2.0-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/mpfr-4.2.0-2-x86_64.pkg.tar.zst -> msys2-mpfr-4.2.0-2-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/libpcre-8.45-3-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/libpcre-8.45-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libpcre-8.45-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libpcre-8.45-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libpcre-8.45-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libpcre-8.45-3-x86_64.pkg.tar.zst -> msys2-libpcre-8.45-3-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/m4-1.4.19-2-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/m4-1.4.19-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/m4-1.4.19-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/m4-1.4.19-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/m4-1.4.19-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/m4-1.4.19-2-x86_64.pkg.tar.zst -> msys2-m4-1.4.19-2-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/perl-5.36.0-1-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/perl-5.36.0-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/perl-5.36.0-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/perl-5.36.0-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/perl-5.36.0-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/perl-5.36.0-1-x86_64.pkg.tar.zst -> msys2-perl-5.36.0-1-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/ncurses-6.4-1-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/ncurses-6.4-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/ncurses-6.4-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/ncurses-6.4-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/ncurses-6.4-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/ncurses-6.4-1-x86_64.pkg.tar.zst -> msys2-ncurses-6.4-1-x86_64.pkg.tar.zst...
-- Downloading https://mirror.msys2.org/msys/x86_64/libcrypt-2.1-4-x86_64.pkg.tar.zst;https://repo.msys2.org/msys/x86_64/libcrypt-2.1-4-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libcrypt-2.1-4-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libcrypt-2.1-4-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libcrypt-2.1-4-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libcrypt-2.1-4-x86_64.pkg.tar.zst -> msys2-libcrypt-2.1-4-x86_64.pkg.tar.zst...
-- Applying patch C:/Users/tgodi/vcpkg/scripts/msys/compile_wrapper_consider_clang-cl.patch
-- Using msys root at C:/Users/tgodi/vcpkg/downloads/tools/msys2/d7266db249278763
-- Generating configure for x86-windows-static
-- Finished generating configure for x86-windows-static
-- Downloading https://mirror.msys2.org/mingw/mingw32/mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst;https://repo.msys2.org/mingw/mingw32/mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/mingw/mingw32/mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/mingw/mingw32/mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/mingw/mingw32/mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst;https://mirror.selfnet.de/msys2/mingw/mingw32/mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst -> mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst...
-- Using cached msys2-msys2-runtime-3.4.6-1-x86_64.pkg.tar.zst.
-- Using msys root at C:/Users/tgodi/vcpkg/downloads/tools/msys2/6f3fa1a12ef85a6f
-- Configuring x86-windows-static-dbg
-- Configuring x86-windows-static-rel
-- Building x86-windows-static-dbg
CMake Warning at scripts/cmake/vcpkg_execute_build_process.cmake:65 (message):
  Please ensure your system has sufficient memory.
Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/gmp/portfile.cmake:94 (vcpkg_install_make)
  scripts/ports.cmake:147 (include)


-- Restarting build without parallelism
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/Users/tgodi/vcpkg/downloads/tools/msys2/d7266db249278763/usr/bin/make.exe -j 7 --trace -f Makefile all
    Working Directory: C:/Users/tgodi/vcpkg/buildtrees/gmp/x86-windows-static-dbg/
    See logs for more information:
      C:\Users\tgodi\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-out.log
      C:\Users\tgodi\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err.log
      C:\Users\tgodi\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-out-1.log
      C:\Users\tgodi\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err-1.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/gmp/portfile.cmake:94 (vcpkg_install_make)
  scripts/ports.cmake:147 (include)




:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsignedcl : Command line warning D9002 : 
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsignedignoring unknown option '-Xcompiler'C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned


C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsignedcl : Command line warning D9002 : C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned
ignoring unknown option '-Xcompiler'
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned
cl : Command line warning D9002 : 
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsignedignoring unknown option '-Xcompiler'C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned

cl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned
cl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned
cl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsignedcl : Command line warning D9002 : ignoring unknown option '-Xcompiler'

cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
gen-trialdivtab.c
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of dataC:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data

C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of dataC:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)

C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
cl : Command line warning D9002 : C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
gen-psqr.c
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)

cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)

cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(349): warning C4018: '>': signed/unsigned mismatch
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(367): warning C4018: '>': signed/unsigned mismatch
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(377): warning C4018: '<': signed/unsigned mismatch
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(251): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(267): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(328): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
LINK : fatal error LNK1104: cannot open file 'm.lib'
LINK : fatal error LNK1104: cannot open file 'm.lib'
LINK : fatal error LNK1104: cannot open file 'm.lib'
```
</details>
<details><summary>C:\Users\tgodi\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err.log</summary>

```
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
make: *** [Makefile:1488: gen-bases.exe] Error 2
make: *** Waiting for unfinished jobs....
make: *** [Makefile:1506: gen-psqr.exe] Error 2
make: *** [Makefile:1494: gen-trialdivtab.exe] Error 2
```
</details>
<details><summary>C:\Users\tgodi\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-out-1.log</summary>

```
Makefile:1467: update target 'fac_table.h' due to: target does not exist
./gen-fac 32 0 >fac_table.h || (rm -f fac_table.h; exit 1)
Makefile:1473: update target 'fib_table.h' due to: target does not exist
./gen-fib header 32 0 >fib_table.h || (rm -f fib_table.h; exit 1)
Makefile:1476: update target 'mpn/fib_table.c' due to: target does not exist
./gen-fib table 32 0 >mpn/fib_table.c || (rm -f mpn/fib_table.c; exit 1)
Makefile:1488: update target 'gen-bases.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-bases.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-bases.c -o gen-bases.exe -lm
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
gen-bases.c
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\Users\tgodi\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
LINK : fatal error LNK1104: cannot open file 'm.lib'
```
</details>
<details><summary>C:\Users\tgodi\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err-1.log</summary>

```
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
make: *** [Makefile:1488: gen-bases.exe] Error 2
```
</details>

**Additional context**

![Screenshot_20](https://github.com/microsoft/vcpkg/assets/28079139/8bce34da-a99c-4c9c-be8c-cb08bb27ccad)

> ~~~
>  LINK : fatal error LNK1104: cannot open file 'm.lib'

Lib `m` shouldn't be used in a MSVC build. The config logs might answer why gmp wants to do it. 
Does this error occur on your machine also for the official x86-windows triplet?

goodnight. today I formatted my computer and reinstalled everything from scratch. now the apparently changed error follows the log.










Package: gmp:x86-windows-static -> 6.2.1#20

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.36.32537.0
-    vcpkg-tool version: 2023-08-02-6d13efa755f9b5e101712d210199e4139b4c29f6
    vcpkg-scripts version: 78b61582c 2023-08-04 (11 hours ago)

**To Reproduce**

`vcpkg install gmp`

**Failure logs**

```
-- Using cached gmp-6.2.1.tar.xz.
-- Cleaning sources at C:/vcpkg/buildtrees/gmp/src/v6.2.1-0c723d4b6f.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/vcpkg/downloads/gmp-6.2.1.tar.xz
-- Applying patch asmflags.patch
-- Applying patch cross-tools.patch
-- Applying patch subdirs.patch
-- Applying patch msvc_symbol.patch
-- Applying patch arm64-coff.patch
-- Applying patch gmp-arm64-asm-fix-5f32dbc41afc.patch
-- Using source at C:/vcpkg/buildtrees/gmp/src/v6.2.1-0c723d4b6f.clean
-- Found external ninja('1.11.0').
-- Getting CMake variables for x86-windows-static
-- Getting CMake variables for x86-windows-static
-- Using cached msys2-autoconf-wrapper-20221207-1-any.pkg.tar.zst.
-- Using cached msys2-automake-wrapper-20221207-1-any.pkg.tar.zst.
-- Using cached msys2-binutils-2.40-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libtool-2.4.7-3-x86_64.pkg.tar.zst.
-- Using cached msys2-make-4.4.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-pkgconf-1.9.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst.
-- Using cached msys2-bash-5.2.015-1-x86_64.pkg.tar.zst.
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst.
-- Using cached msys2-file-5.44-5-x86_64.pkg.tar.zst.
-- Using cached msys2-gawk-5.2.1-2-x86_64.pkg.tar.zst.
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst.
-- Using cached msys2-gzip-1.12-2-x86_64.pkg.tar.zst.
-- Using cached msys2-diffutils-3.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.6-1-x86_64.pkg.tar.zst.
-- Using cached msys2-autoconf2.71-2.71-1-any.pkg.tar.zst.
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst.
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libintl-0.21-2-x86_64.pkg.tar.zst.
-- Using cached msys2-zlib-1.2.13-1-x86_64.pkg.tar.zst.
-- Using cached msys2-tar-1.34-3-x86_64.pkg.tar.zst.
-- Using cached msys2-findutils-4.9.0-3-x86_64.pkg.tar.zst.
-- Using cached msys2-gmp-6.2.1-2-x86_64.pkg.tar.zst.
-- Using cached msys2-gcc-libs-11.3.0-3-x86_64.pkg.tar.zst.
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst.
-- Using cached msys2-liblzma-5.4.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libzstd-1.5.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libreadline-8.2.001-3-x86_64.pkg.tar.zst.
-- Using cached msys2-mpfr-4.2.0-2-x86_64.pkg.tar.zst.
-- Using cached msys2-libpcre-8.45-3-x86_64.pkg.tar.zst.
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst.
-- Using cached msys2-perl-5.36.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-ncurses-6.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libcrypt-2.1-4-x86_64.pkg.tar.zst.
-- Using msys root at C:/vcpkg/downloads/tools/msys2/d7266db249278763
-- Generating configure for x86-windows-static
-- Finished generating configure for x86-windows-static
-- Using cached mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.6-1-x86_64.pkg.tar.zst.
-- Using msys root at C:/vcpkg/downloads/tools/msys2/6f3fa1a12ef85a6f
-- Configuring x86-windows-static-dbg
-- Configuring x86-windows-static-rel
-- Building x86-windows-static-dbg
CMake Warning at scripts/cmake/vcpkg_execute_build_process.cmake:65 (message):
  Please ensure your system has sufficient memory.
Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/gmp/portfile.cmake:94 (vcpkg_install_make)
  scripts/ports.cmake:147 (include)


-- Restarting build without parallelism
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/vcpkg/downloads/tools/msys2/d7266db249278763/usr/bin/make.exe -j 7 --trace -f Makefile all
    Working Directory: C:/vcpkg/buildtrees/gmp/x86-windows-static-dbg/
    See logs for more information:
      C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-out.log
      C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err.log
      C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-out-1.log
      C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err-1.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/gmp/portfile.cmake:94 (vcpkg_install_make)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-out.log</summary>

```
Makefile:1470: update target 'gen-fac.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-fac.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-fac.c -o gen-fac.exe
Makefile:1479: update target 'gen-fib.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-fib.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-fib.c -o gen-fib.exe
Makefile:1488: update target 'gen-bases.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-bases.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-bases.c -o gen-bases.exe -lm
Makefile:1494: update target 'gen-trialdivtab.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-trialdivtab.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-trialdivtab.c -o gen-trialdivtab.exe -lm
Makefile:1500: update target 'gen-jacobitab.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-jacobitab.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-jacobitab.c -o gen-jacobitab.exe
Makefile:1506: update target 'gen-psqr.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-psqr.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-psqr.c -o gen-psqr.exe -lm
Makefile:830: update target 'config.h' due to: stamp-h1
test -f config.h || rm -f stamp-h1
gen-fib.c
gen-fac.c
test -f config.h || /usr/bin/make  stamp-h1
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned
gen-jacobitab.c
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsignedcl : Command line warning D9002 : ignoring unknown option '-Xcompiler'


C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsignedcl : Command line warning D9002 : 
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsignedignoring unknown option '-Xcompiler'
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
gen-trialdivtab.c
cl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of dataC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data


C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'C
:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned

cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
gen-bases.c
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned

cl : Command line warning D9002 : Cignoring unknown option '-Xcompiler':\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned

cl : Command line warning D9002 : C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsignedignoring unknown option '-Xcompiler'

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned
cl : Command line warning D9002 : Cignoring unknown option '-Xcompiler':\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned

cl : Command line warning D9002 : C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsignedignoring unknown option '-Xcompiler'

cl : Command line warning D9002 : C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsignedignoring unknown option '-Xcompiler'

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned

cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned

cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
Ccl : Command line warning D9002 : :\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsignedignoring unknown option '-Xcompiler'

cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsignedcl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned

gen-psqr.c
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsignedC:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data

C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-trialdivtab.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c)
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(349): warning C4018: '>': signed/unsigned mismatch
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(367): warning C4018: '>': signed/unsigned mismatch
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(377): warning C4018: '<': signed/unsigned mismatch
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(251): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(267): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
./../src/v6.2.1-0c723d4b6f.clean/gen-psqr.c(328): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
LINK : fatal error LNK1104: cannot open file 'm.lib'
LINK : fatal error LNK1104: cannot open file 'm.lib'
LINK : fatal error LNK1104: cannot open file 'm.lib'
```
</details>
<details><summary>C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err.log</summary>

```
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'cl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'cl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'cl : Command line warning D9002 : 
ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
make: *** [Makefile:1506: gen-psqr.exe] Error 2
make: *** Waiting for unfinished jobs....
make: *** [Makefile:1488: gen-bases.exe] Error 2
make: *** [Makefile:1494: gen-trialdivtab.exe] Error 2
```
</details>
<details><summary>C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-out-1.log</summary>

```
Makefile:1467: update target 'fac_table.h' due to: target does not exist
./gen-fac 32 0 >fac_table.h || (rm -f fac_table.h; exit 1)
Makefile:1473: update target 'fib_table.h' due to: target does not exist
./gen-fib header 32 0 >fib_table.h || (rm -f fib_table.h; exit 1)
Makefile:1476: update target 'mpn/fib_table.c' due to: target does not exist
./gen-fib table 32 0 >mpn/fib_table.c || (rm -f mpn/fib_table.c; exit 1)
Makefile:1488: update target 'gen-bases.exe' due to: target does not exist
compile cl.exe -DWIN32 -D_WINDOWS -D_DEBUG -Xcompiler -nologo -Xcompiler -W3 -Xcompiler -utf-8 -Xcompiler -MP -Xcompiler -MTd -Xcompiler -Z7 -Xcompiler -Ob0 -Xcompiler -Od -Xcompiler -RTC1 `test -f 'gen-bases.c' || echo './../src/v6.2.1-0c723d4b6f.clean/'`gen-bases.c -o gen-bases.exe -lm
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
gen-bases.c
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(717): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(758): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(976): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1012): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1068): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1215): warning C4244: 'initializing': conversion from 'const mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1243): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1247): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1472): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1478): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1545): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(1839): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2017): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(2635): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3350): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3370): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3791): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3792): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3793): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3863): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3864): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3865): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3936): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3937): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(3938): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4039): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4102): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4120): warning C4146: unary minus operator applied to unsigned type, result still unsigned (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4546): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
C:\vcpkg\buildtrees\gmp\src\v6.2.1-0c723d4b6f.clean\mini-gmp/mini-gmp.c(4559): warning C4244: '=': conversion from 'mp_limb_t' to 'unsigned char', possible loss of data (compiling source file ./../src/v6.2.1-0c723d4b6f.clean/gen-bases.c)
LINK : fatal error LNK1104: cannot open file 'm.lib'
```
</details>
<details><summary>C:\vcpkg\buildtrees\gmp\build-x86-windows-static-dbg-err-1.log</summary>

```
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
cl : Command line warning D9002 : ignoring unknown option '-Xcompiler'
make: *** [Makefile:1488: gen-bases.exe] Error 2
```
</details>

**Additional context**

--- a/ports/gmp/portfile.cmake
+++ b/ports/gmp/portfile.cmake
@@ -35,6 +35,7 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     vcpkg_list(APPEND OPTIONS
         "ac_cv_func_memset=yes"
         "gmp_cv_asm_w32=.word"
+        "gmp_cv_check_libm_for_build=no"
     )
 endif()
 

--- a/ports/gmp/vcpkg.json
+++ b/ports/gmp/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "gmp",
   "version": "6.2.1",
-  "port-version": 20,
+  "port-version": 21,
   "description": "The GNU Multiple Precision Arithmetic Library",
   "homepage": "https://gmplib.org",
   "license": "LGPL-3.0-only OR GPL-2.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2946,7 +2946,7 @@
     },
     "gmp": {
       "baseline": "6.2.1",
-      "port-version": 20
+      "port-version": 21
     },
     "gmsh": {
       "baseline": "4.9.0",

--- a/versions/g-/gmp.json
+++ b/versions/g-/gmp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "726604ff590362102d805e16508ae20730583101",
+      "version": "6.2.1",
+      "port-version": 21
+    },
+    {
       "git-tree": "5925ffa7bf1165b7ca3144e4253a19dcfc47a5ad",
       "version": "6.2.1",
       "port-version": 20

